### ISSUE TITLE
laminarFlatPlateLTS has multiple file naming issues

### ISSUE THREAD

**Issue by jg110** [vincentcasseau/hyStrath#8]
There are three file naming issues within the laminarFlatPlateLTS example:

- The log file is named `log.hyFoam`, but the gnuplot scripts are looking for `log.hyFoamLTS`
- The `Alltest` script looks for `line_p_rho_Tt.xy` (line 16), but the corresponding file in postProcessing/singleGraph/5e-05 is `line_Tt_p_rho.xy`. Changing the name in `Alltest` still results in the script failing because the corresponding file in `gnuplot/solution/singleGraph/5e-05` is named `line_p_rho_Tt.xy`.
- The `Alltest` script also looks for `log.hy2Foam` (line 60). When I change `log.hy2Foam` to `log.hyFoam`, I get an awk syntax error: 

```
TEST FAILED: line_Tt_p_rho.xy, line 175
awk: cmd. line:1: /^Time =/,/^Time =/ {lastc = $0; next}{ if ( lastc != \"\") { print lastc; lastc = \"\"; } print }
awk: cmd. line:1:                                                       ^ backslash not last character on line
awk: cmd. line:1: /^Time =/,/^Time =/ {lastc = $0; next}{ if ( lastc != \"\") { print lastc; lastc = \"\"; } print }
awk: cmd. line:1:                                                       ^ syntax error
TEST FAILED: logIntegratedWallHeatFlux
```

**Comment by vincentcasseau** [comment]
Thanks for your rigour @jg110. I hope that I have addressed all your comments in the commit 4d79218.

**Comment by jg110** [comment]
Thanks, after testing, the script works.
